src/fu-security-attr.c

```diff
@@ -247,3 +250,61 @@ fu_security_attr_get_result(FwupdSecurityAttr *attr)
 	/* TRANSLATORS: Suffix: the fallback HSI result */
 	return _("Failed");
 }
+
+gchar *
+fu_security_attrs_to_json_string(FuSecurityAttrs *attrs)
+{
+	/*  Expected Seccurity attr format
+	{
+	    "SecurityAttrs": {
+		"Attrs": [
+		    {
+			"name": "aaa",
+			"value": "bbb"
+		    }
+		]
+	    }
+	}
```

Comment:
I think this should be added as a documentation comment for the function explaining this information, or linking elsewhere.

Reply:
I'll move them before the function and describe the function and JSON format.